test/deploy/proxy.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai';
+import { ethers, upgrades } from 'hardhat';
+
+describe('Proxy', function () {
+  it('FleekERC721', async () => {
+    const Contract = await ethers.getContractFactory('FleekERC721');
+    const ContractV2 = await ethers.getContractFactory('FleekERC721');
+
+    const instance = await upgrades.deployProxy(Contract, [
+      'Collection Name',
+      'SYMBOL',
+    ]);
+    const upgraded = await upgrades.upgradeProxy(instance.address, ContractV2);
```

Comment:
LGTM. I think a good addition to this proxy test; would be to change some state on the contract, and then upgrade, and then check the state is still changed after upgrade

Reply:
This file may be removed since it is kind of testing the plugin and not our deployment script, this change is already addressed in #83.